src/api/userApi.ts

```diff
@@ -1,5 +1,5 @@
-import { CognitoUserSession } from 'amazon-cognito-identity-js';
 import { Auth } from 'aws-amplify';
+import { CognitoUserSession } from 'amazon-cognito-identity-js';
```

Comment:
skjønner ikke helt dette? du importerer den allikevel fra pakken som du har fjernet?

Reply:
den ligger nå som en egen modul som kommer med via `aws-amplify`: https://github.com/amazon-archives/amazon-cognito-identity-js#amazon-cognito-identity-sdk-for-javascript

Reply:
ahh ok 👍